core/src/main/kotlin/com/justai/jaicf/activator/strategy/ActivationByConfidence.kt

```diff
@@ -0,0 +1,39 @@
+package com.justai.jaicf.activator.strategy
+
+import com.justai.jaicf.context.BotContext
+import com.justai.jaicf.model.activation.Activation
+import com.justai.jaicf.model.activation.ActivationStrategy
+import com.justai.jaicf.model.state.StatePath
+
+/**
+ * Default implementation of [ActivationStrategy], used by [com.justai.jaicf.BotEngine]
```

Comment:
**Should be fixed**
Let's remove **Default** word from this doc. 
The info about default implementation must be documented in `BotEngine` and not inside this concrete implementation.